lib/ex_css_modules/ex_css_modules.ex

```diff
@@ -108,12 +106,20 @@ defmodule ExCSSModules do
 
     iex> class_name(%{"hello" => "world", "foo" => "bar"}, [{"hello", true}, {"foo", false}])
     "world"
+
+    iex> class_name(%{"hello" => "world", "foo" => "bar"}, [{"hello", false}])
+    nil
   """
   def class_name(definition, keys) when is_list(keys) do
-    keys
-    |> Enum.map(&class_name(definition, &1))
-    |> Enum.reject(&is_nil/1)
-    |> Enum.join(" ")
+    list = keys
+      |> Enum.map(&class_name(definition, &1))
+      |> Enum.reject(&is_nil/1)
+
+    if Enum.any?(list) do
```

Comment:
Prefer to away from if statements as much as possible, ie:
```ex
defp join_class_name(list) when length(list) == 0, do: nil
defp join_class_name(list), do: Enum.join(list, " ")
```

Reply:
This way we can just keep piping the values as well.

Reply:
Kinda had the same thought, so you really think this is more clear?

Reply:
I don't know yet ;) it is quite clear when you read the `join_class_name` function what it does, it gets rid of an if statement and it allows you to keep piping the value to the next function which I like a lot.